src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -89,6 +90,7 @@ export const ConfirmSummary = ({
   const buyAssetFeeAsset = useAppSelector(state =>
     selectFeeAssetById(state, buyAsset?.assetId ?? ''),
   )
+  const isAccountLoading = useAppSelector(selectIsAccountLoading)
```

Comment:
blocking: plural accounts vernacular, or alternatively `isAnyAccountIdsLoading`
```suggestion
  const isAccounstLoading = useAppSelector(selectIsAccountLoading)
```